--- pinctrl-mlxbf3-1.0/pinctrl-mlxbf3.c	2024-11-18 08:43:02.000000000 -0500
+++ ../pinctrl-mlxbf3-1.0/pinctrl-mlxbf3.c	2025-01-23 17:10:59.376674071 -0500
@@ -29,21 +29,6 @@
 #endif
 #endif
 
-#ifdef DEFINE_PIN_MACROS
-struct pinfunction {
-	const char *name;
-	const char * const *groups;
-	size_t ngroups;
-};
-
-#define PINCTRL_PINFUNCTION(_name, _groups, _ngroups)	\
-((struct pinfunction) {					\
-		.name = (_name),			\
-		.groups = (_groups),			\
-		.ngroups = (_ngroups),			\
-	})
-#endif
-
 #define MLXBF3_NGPIOS_GPIO0    32
 #define MLXBF3_MAX_GPIO_PINS   56
 
